src/app/inlineinput/inlineinput.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="inlineinput-container"
+  [class.read-only]="readOnly">
+  <textarea
```

Comment:
If this is an inline input, why is it a `<textarea>`?

Reply:
The title can be big and may have adjusted height, thus the text area is used.